app/assets/stylesheets/components/page_specific/_promo.scss

```diff
@@ -9,19 +9,73 @@
 .promo__heading {
   @extend %font-heading-medium;
   @include body(18, 24);
-  margin-top: 0;
+  margin: 0 0 $baseline-unit;
```

Comment:
I don't know if we explicitly state this anywhere to be honest. I think in this case I agree, its simpler to just use `margin-bottom: 0;` but I wouldn't say its a hard and fast rule not to use the 3 args.